eng/versioning/version_client.txt

```diff
@@ -400,8 +398,7 @@ com.azure.tools:azure-sdk-build-tool;1.0.0-beta.1;1.0.0-beta.2
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
 unreleased_com.azure:azure-core;1.35.0-beta.1
-unreleased_com.azure:azure-identity-providers-jdbc-mysql;1.0.0
-unreleased_com.azure:azure-identity-providers-jdbc-postgresql;1.0.0
+unreleased_com.azure:azure-identity-extensions;1.0.0
```

Comment:
Do we still use this now?

Reply:
No, but I will use another PR to address this and other issues.

Reply:
Ok